fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStoreBase.java

```diff
@@ -1725,6 +1726,23 @@ default int getPipelineSize(@Nonnull PipelineOperation pipelineOperation) {
         return plan.execute(this, EvaluationContext.EMPTY, continuation, executeProperties);
     }
 
+    /**
+     * Execute a query.
+     * @param plan the plan to execute
+     * @param continuation continuation from a previous execution of this same plan
+     * @param evaluationContext a context for the execution of this plan
+     * @param executeProperties limits on execution
+     * @return a cursor for query results of type {@link QueryResult}
+     * @see RecordQueryPlan#execute
+     */
+    @Nonnull
+    default RecordCursor<QueryResult> executePlan(@Nonnull RecordQueryPlan plan,
```

Comment:
The other overloads are called `executeQuery`.

Reply:
done